iina/Utility.swift

```diff
@@ -79,6 +79,21 @@ class Utility {
       return false
     }
   }
+  
+  static func quickSavePanel(title: String, types: [String], ok: (URL) -> Void) -> Bool {
+    let panel = NSSavePanel()
+    panel.title = title
+    panel.canCreateDirectories = true
+    panel.allowedFileTypes = types
+    if panel.runModal() == NSFileHandlingPanelOKButton {
+      if let url = panel.url {
+        ok(url)
+      }
+      return true
```

Comment:
For these panels, we don't check the return value anyway. Checking url's non-emptiness seems to be done all in the callbacks.